src/screens/Library.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { useState, useEffect } from 'react';
+
+import Card from "../components/Card";
+import Loader from "../components/Loader";
+import Button from "../components/Button";
+
+import getDataFromApi from '../utils';
+
+
+const Library = (props) => {
+    // Toggle State
+    const [toggle, updateToggle] = useState(false);
+
+    // API URL by Toggle State
+    const apiUrl = toggle ?
+        "https://match-labs-api.herokuapp.com/api/no_auth/candidates" :
+        "https://match-labs-api.herokuapp.com/api/no_auth/labs";
+    const [jsonData, updateData] = useState([]);
+    const [loading, updateLoading] = useState(true);
+
+
+    // Request Data from API on Toggle State Change
+    useEffect(() => {
+        const stopLoading = getDataFromApi(apiUrl, updateData, () => { });
+        updateLoading(stopLoading);
+    }, [toggle]);
+
+
+    // Set Card Data by Card Type
+    const card = {}
+
+    const setCardData = (data) => {
```

Comment:
It is great that you do data normalisation here, but one thing you could improve is do it directly in the `utils.js` method!